platform/android/export/export.cpp

```diff
@@ -847,6 +847,135 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 					uint32_t name = decode_uint32(&p_manifest[iofs + 12]);
 					String tname = string_table[name];
 
+					int dof_index = p_preset->get("graphics/degrees_of_freedom"); // 0: none, 1: 3dof and 6dof, 2: 6dof
+
+					if (tname == "uses-feature" && dof_index > 0) {
```

Comment:
This should also check if 'Oculus Mobile VR' is selected as Xr mode, since that tag is only valid for Oculus devices at the moment.

Reply:
Rather than silently not adding the tag if it's not an oculus mobile device, I was hoping that in the godot_oculus_mobile docs it'd just specify to have the dropdown selected to the 6dof option if targeting the oculus quest.

Reply:
One of the most important UX learning is that typical users never read the documentation, and so, if a feature has the potential to have a negative impact, then that feature should be placed behind a check to ensure that the feature is not accidentally enabled by the user.

This applies here as well since based on the [Google docs](https://developers.google.com/vr/develop/android/3dof-to-6dof) the tag **must** not be included for apps with no 6DoF features.

As you mentioned, we'll include a description of the dropdown's purpose in the docs, but a check is also needed in case the user doesn't read the doc or forget about it.

Reply:
I've added a check then a warning print if the user forgets to set the xr mode.